build.gradle

```diff
@@ -405,6 +406,19 @@ task integTestRemote(type: RestIntegTestTask) {
     }
 }
 
+
+// test retry configuration
+subprojects {
+    apply plugin: "org.gradle.test-retry"
+    tasks.withType(Test).configureEach {
+        retry {
+            failOnPassedAfterRetry = false
+            maxRetries = 3
+            maxFailures = 10
```

Comment:
Let's make this a lower number.  Typically we only should see 1 or 2 failures getting retried in a test run, so this value should definitely be less than the max # of tests we run (7) and probably closer to 3.

Reply:
Changed it to 1 and 3 respectively.

Reply:
Looks like 1 and 3 didn't work. Will try incrementing `maxRetries.`

Reply:
Incremented it to 3 and 5.